src/components/thumbnail/tab-panel-documents-section.tsx

```diff
@@ -79,25 +79,27 @@ function getSectionDocs(section: NavTabSectionModelType, stores: IStores, classS
   return sectDocs;
 }
 
-export const TabPanelDocumentsSection = observer(({ tab, section, index, numOfSections, stores, scale, selectedDocument,
+export const TabPanelDocumentsSection = observer(({ tab, section, index, numSections, stores, scale, selectedDocument,
                                   onSelectNewDocument, onSelectDocument, onDocumentDragStart,
                                   onDocumentStarClick, onDocumentDeleteClick }: IProps) => {
     const { user } = stores;
     const showNewDocumentThumbnail = section.addDocument && !!onSelectNewDocument;
     const newDocumentLabel = getNewDocumentLabel(section, stores);
-    const numPanels = user.isNetworkedTeacher ? numOfSections + 1 : numOfSections;
+    const numPanels = user.isNetworkedTeacher ? numSections + 1 : numSections;
     const isTopPanel = index === 0 && numPanels > 1;
-    const isBottomPanel = index === numOfSections - 1 && index > 0;
-    const tabName = tab.toLowerCase().replace(' ', '-');
+    const isBottomPanel = index === numSections - 1 && index > 0;
+    const isSinglePanel = numSections < 2;
```

Comment:
My previous comment about not having to handle the `undefined` case was specifically about the redundant test for `numSections` in `isSinglePanel = numSections && numSections > 2`. The only reason to use the double-test is if you have to handle `undefined` so I assumed that's what you were doing. In the three lines above this (`numPanels`, `isTopPanel`, `isBottomPanel`) you _do_ still need to be concerned about the `0` case, so the question is: after removing the tests for `numSections` do these other definitions still work for the `numSections === 0` case? If not, we could have a baby-thrown-out-with-the-bath-water situation. 😉

Reply:
I don't think so. In line 89, if numSections=0, the only possible values for numPanels is 0 || 1, neither of which will pass `numPanels > 1`. In line 90, I don't think we'll ever have an index<0. The `undefined` case may just be a remnant of my experiments. Current code also does not have the test.

Reply:
Right, but in line 88 the previous (now removed) version of the code was:
```
    const numPanels = numSections && user.isNetworkedTeacher ? numSections + 1 : numSections;
```
which says that networked teachers only get an additional panel if there's already at least one section. I suspect the older code never had to deal with the `numSections === 0` case, so the distinction didn't matter. So my question is, should networked teachers with no sections have a panel or not?